tgui/packages/tgui/hotkeys.js

```diff
@@ -114,21 +148,25 @@ const handlePassthrough = (e, eventType) => {
   }
   const keyData = getKeyData(e);
   const { keyCode, ctrlKey, shiftKey } = keyData;
+  const byondKey = keyCodeToByond(keyCode);
   // NOTE: We pass through only Alt of all modifier keys, because Alt
   // modifier (for toggling run/walk) is implemented very shittily
```

Comment:
Actually
```suggestion
  // modifier is implemented very shittily
```

This comment might need updating somewhat like this. Perhaps it would be even better to explain what is going on with the block below this at 156, AND with `if (ctrlKey || shiftKey)` down on line 163 too. Since they sorta do similar things but also are different.